custom_connectors/custom_auth/eventbrite_connector.rb

```diff
@@ -0,0 +1,104 @@
+{
+  title: "Eventbrite (custom)",
+
+  connection: {
+    fields: [
+      {
+        name: "personal_token",
+        control_type: "password",
+        hint: "Find your personal token here: https://www.eventbrite.com/myaccount/apps/",
```

Comment:
Can we do basic auth? Also, can we have comments where we point to the actual documentation on Eventbrite?

Reply:
No basic auth for Eventbrite